src/components/Explore/utils.ts

```diff
@@ -12,7 +12,7 @@ import {
 } from 'lodash';
 import { color } from 'd3-color';
 import { schemeCategory10 } from 'd3-scale-chromatic';
-import { fetchProjections } from 'common/utils/model';
+import { fetchProjectionsRegion } from 'common/utils/model';
```

Comment:
FWIW- "fetchProjections" is a very antiquated name.  I didn't get a chance to review the PR that added fetchProjectionsRegion, but we would ideally be moving towards better naming, e.g. `fetchRegionData()` or `fetchLocationData()`. We can punt for now though.

Reply:
`fetchRegionTimeseries` might be good too

Reply:
Actually, i'd rather change the name alongside changing the `Projections` class name, will 🦶 🏈 for now

Reply:
SG.  I'd keep "timeseries" out of it since it includes a bunch of timeseries and non-timeseries data.

FWIW- It also includes helper methods for getting risk level color, etc. so it's really quite broad. 😁